### PR TITLE
「カルヂナルのコンポーネンツ エーピーアイ 」三世へのアップデート 

### PR DESCRIPTION
**Pull request by Pyrofab**
fucking weeb

### CODE REVIEW

File: src/main/java/tk/valoeghese/tknm/common/InnateAbilityManager.java [hunk 1]
```diff
@@ -46,7 +47,8 @@ public void fromTag(CompoundTag tag) {
 
 		@Nullable
 		public Ability<?> provideInnateAbility(UUID uuid, Random rand) {
-			this.sync();
+			// FIXME sync requires a MinecraftServer, but is it really needed?
+//			LevelComponents.sync(INNATE_ABILITY_COMPONENT, server);
```
Comment (Pyrofab) [inline src/main/java/tk/valoeghese/tknm/common/InnateAbilityManager.java]: Fun fact, the old `LevelSyncedComponent#sync` was a no-op... So either there is a bug to fix, which can be fixed by passing a MinecraftServer to this method somehow, or there is no bug and this line can be removed.

**Comment by valoeghese** [comment]
can you clean up the PR so I can see what you actually change

File: src/main/java/tk/valoeghese/tknm/common/InnateAbilityManager.java (hunk 1 above)
Comment (valoeghese) [reply src/main/java/tk/valoeghese/tknm/common/InnateAbilityManager.java]: should have @deprecated the method

**Review by valoeghese — COMMENTED** [review]
seems fine but can you clean up your patches to InnateAbilityManager so I can see what you change

File: src/main/java/tk/valoeghese/tknm/common/InnateAbilityManager.java (hunk 1 above)
Comment (Pyrofab) [reply src/main/java/tk/valoeghese/tknm/common/InnateAbilityManager.java]: Indeed, but it's a bit too late now 🤷‍♀️

**Comment by valoeghese** [comment]
Cardinal Components v3 code is bloat mess :tiny potato:

**Review by valoeghese — APPROVED** [review]

**Pull request merged**